content/installation/image-builder.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: bt_wiki
+title: Using the Image Builder
+category: Installation
+draft: false
+weight: 600
+
+---
+
+{{% gsNote title="Supported Clouds" %}}
+ * AWS
+ * OpenStack
+ * vSphere
+{{% /gsNote %}}
+
+{{% gsNote title="Prerequisites" %}}
+ * [packer](https://www.packer.io/intro/getting-started/setup.html) += 0.8.6 (older versions not tested)
+ * Account credentials for each cloud platform you want to build on.
```

Comment:
I think I'd like to present this as tabs for each platform if possible (e.g. the Scss/SASS examples on http://sass-lang.com/guide ). Anyone know how easy that is to achieve with the docs.getcloudify.org framework?